### ISSUE TITLE
Does the data have to be labeled？

### ISSUE THREAD

**Issue by huangxin4520** [whyre788/GAN-1D#1]
If I want to generate fault information, do I need to have a fault tag?

**Comment by whyre788** [comment]
no, dont need any label, but im still testing this algorithm, plz wait few days

**Comment by heheda5hao** [comment]
Excuse me, can you give me an annotated program code? Because there are some things I don't quite understand.Thank you very much!!!
